src/chrome/content/rules/ComputerWoche.de.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="ComputerWoche">
+    <target host="*.computerwoche.de" />
+
+    <rule from="^http://(images|www)\.computerwoche\.de/"
+        to="https://$1.computerwoche.de/" />
```

Comment:
Some content used to work in the past, but at the moment you are right. No content on the www subdomain is served over tls for now.
I removed it.